.github/workflows/CI.yml

```diff
@@ -65,5 +51,4 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
 
     - name: Run tests
-      run: |
-        pytest
+      run: pytest -v
```

Comment:
why -v? It spams the logs full usually while without -v it only prints them in case of error

Reply:
Fixed https://github.com/torchmd/torchmd-net/pull/53/commits/60e652e09fac05638a6514f81518aa1b07ba2981